sqladmin/application.py

```diff
@@ -131,7 +132,31 @@ class UserAdmin(ModelView, model=User):
             )
             view.async_engine = True
 
-        self._views.append((view()))
+        view_instance = view()
+        funcs = inspect.getmembers(view_instance, predicate=inspect.ismethod)
```

Comment:
This is doing something very similar to `expose` we should probably see if there's a chance to to re-use any of that, I'll take a look too.

Reply:
Yeah maybe we can refactor it, maybe after this one reviewed/merged